### PR TITLE
Qa pipeline num spans

### PR DESCRIPTION
**Pull request by kylesayrs**
This PR adds support for a max_num_spans argument to the QA pipeline. When the given context is too long, the inputs will be split across multiple spans. This argument limits the maximum number of spans that will be fed to the model.

### CODE REVIEW

**Review by rahul-tuli — DISMISSED** [review]

File: src/deepsparse/transformers/pipelines/question_answering.py [hunk 1]
```diff
@@ -118,12 +118,13 @@ class QuestionAnsweringPipeline(TransformersPipeline):
...
-        controls the size of that overlap. Currently, only reading the first span
-        is supported (everything after doc_stride will be truncated). Default
-        is 128
-    :param max_question_len: maximum length of the question after tokenization.
+        controls the size of that overlap. Default is 128
+    :param max_question_length: maximum length of the question after tokenization.
         It will be truncated if needed. Default is 64
-    :param max_answer_len: maximum length of answer after decoding. Default is 15
+    :param max_answer_length: maximum length of answer after decoding. Default is 15
+    :param max_num_spans: if the context is too long to fit with the question for the
```
Comment (bfineran) [inline src/deepsparse/transformers/pipelines/question_answering.py:125]: let's rename to just `num_spans` to be consistent with the transformers repo

**Review by bfineran — CHANGES_REQUESTED** [review]
Looks great @kylesayrs pending name change. Let's run this through `deepsparse.transformers.eval_downstream` for a QA task and model to make sure nothing broke with this

**Comment by kylesayrs** [comment]
python3 eval_downstream.py zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none --dataset squad --max-samples 150

{'exact_match': 85.52631578947368, 'f1': 90.95891903672225}

**Review by bfineran — APPROVED** [review]

**Review by rahul-tuli — APPROVED** [review]
GG Kyle!

**Pull request merged**